docs/NSoT-CentOS-Install.rst

```diff
@@ -0,0 +1,67 @@
+#############
+NSoT on CentOS Install Instructions 
+#############
+
+This installation guide assumes that you have installed CentOS 12.4 on your machine, and are wanting to install NSoT.
+This guide will help you install NSoT and then run it locally from a browser window.
+
+
+1. Install Python, Dependencies, PIP, and NSOT
+==============
+
+To ensure your CentOS installation is up to date, please update it. 
+Once complete, open a command prompt and run the following:
+
+          $ sudo yum install -y   nodejs mod_ssl openssl git
+          
+          $ sudo yum install -y   python-devel
+          
+          $ sudo yum install -y  libffi-devel openssl-devel
+          
+          $ sudo yum install -y  epel-release
+          
+          $ sudo yum install -y  python-pip
+          
```

Comment:
Fixed with "code" text should be preceded by a `.. code:: bash` followed by a blank line. And then the text should be indented 4 spaces.